Update 'bmc datetime ntpconfig' processing

Smith-x86 opened the pull request:
After adding 'bmc syslog' commands to netconfig the 'bmc datetime ntpconfig' command doesn't work correctly ("Invalid argument" message is printed).

This commit fixed that bug.

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

Smith-x86 committed:
Update 'bmc datetime ntpconfig' processing

After adding 'bmc syslog' commands to netconfig the 'bmc datetime
ntpconfig' command doesn't work correctly ("Invalid argument"
message is printed).

This commit fixed that bug.

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

nest1ing commented on src/netconfig.cpp:
```diff
@@ -82,7 +82,7 @@ static void cmdHostname(Dbus& bus, Arguments& args)
 }
 
-/** @brief Set default gateway: `getway IP` */
+/** @brief Set default gateway: `gateway IP` */
```
I would prefer if You fix this by a separate commit in this pull-request.

Smith-x86 replied:
Done

nest1ing commented on src/netconfig.cpp:
```diff
@@ -322,6 +322,7 @@ static void cmdVlan(Dbus& bus, Arguments& args)
     puts(completeMessage);
 }
 
+/** @brief Configure remote syslog server: `set ADDR[:PORT]` */
```
This comment and the such below also don't seem to me relevant for the commit message.

Smith-x86 replied:
Done

nest1ing reviewed (commented):
Looks good to me except mixing of the different changes into a single commit.
Could You split them?

Smith-x86 committed:
Fixed a typo in the cmdGateway() description

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

Smith-x86 committed:
Added some methods descriptions

Added descriptions for cmdSyslogSet(), cmdSyslogReset() and
cmdSyslogShow() methods

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>
